.github/workflows/ci.yml

```
@@ -95,6 +95,26 @@ jobs:
           CARGO_INCREMENTAL: 1
           RUST_BACKTRACE: 1
 
+  run_ignored_tests:
+    name: Run Specific Ignored Tests in Release Mode
```

nit: I'd just make the job name a tad more explicit, something like this?
```suggestion
    name: Slow evm_arithmetization tests in release mode
```

---

Done